src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netcoreapp1.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PrecompileContentRoot Condition="'$(PrecompileContentRoot)'==''">$(MSBuildProjectDirectory)</PrecompileContentRoot>
+    <PrecompileOutputPath Condition="'$(PrecompileOutputPath)'==''">$(OutputPath)</PrecompileOutputPath>
+  </PropertyGroup>
+  <Target Name="Precompile"
+    AfterTargets="Publish"
```

Comment:
We should play around with this to make it work with both build and publish (perhaps two separate targets which call a third). It's probably a common scenario for users to want to try out precompilation in a non-published app. Additionally, we don't actually require the application to be published, just built to be able to run the tool.